_docs_integrate/read-attributes-from-peer.md

```diff
@@ -131,7 +131,21 @@ If the Recipient does not want the Sender to read any of its Attributes and, the
+If the Recipient agrees to share a requested Attribute with the Sender, it can accept the associated ReadAttributeRequestItem contained in the Request for reading Attributes. In particular, it must then provide the requested Attribute for its Response to the Request. Depending on whether the Recipient wants to share an Attribute that already exists as a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) or that has to be created first, different parameters for this must be used. These [AcceptReadAttributeRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#acceptreadattributerequestitemparameters) are described in the corresponding section of the Data Model Overview.
+
+Accepting a ReadAttributeRequestItem with a new Attribute or an existing, that wasn't shared with the Sender before, leads to the creation of a LocalAttribute with a [LocalAttributeShareInfo]({% link _docs_integrate/data-model-overview.md %}#localattributeshareinfo) contained within its `shareInfo` property, whose underlying `content` is given by the shared Attribute. An appropriate AcceptResponseItem of type [ReadAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#readattributeacceptresponseitem) is generated, which incorporates the `id` of the LocalAttribute with the LocalAttributeShareInfo in its `attributeId` property and the shared Attribute in its `attribute` property.
```

Comment:
Also emphazise that no predecessor of the existing Attribute was shared with the peer when a ReadAttributeAcceptResponseItem is generated.